recipe/meta.yaml

```diff
@@ -24,6 +26,7 @@ requirements:
     - {{ stdlib("c") }}                # [not win]
     - {{ compiler('m2w64_c') }}        # [win]
     - {{ stdlib("m2w64_c") }}          # [win]
+    - mingw-w64-ucrt-x86_64-winpthreads-git  # [win]
```

Comment:
Likely unnecessary - `libwinpthread` will automatically get pulled in and exported into run requirements. For example, see https://github.com/conda-forge/r-protolite-feedstock/pull/44.

```suggestion
```